### ISSUE TITLE
[Accessibility]: Commit page - Commit detail buttons have no descriptive labels

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#34972]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34423

### Problem description

Buttons:
<img width="382" alt="image" src="https://user-images.githubusercontent.com/9516420/166908498-01524a61-426e-4f90-9a60-831438406585.png">

These buttons do not have any labels, so screen reader users are unable to accurately understand what they do.

### Expected behavior

All buttons have appropriate labels, we should use the same value as the tooltip that is on each label.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
Fixed through https://github.com/sourcegraph/sourcegraph/pull/34960
